### PR TITLE
Add setup script

### PR DESCRIPTION
**Pull request by Lumene98**
Take a look at it, I didn't know exactly where to put it(maybe it's better in the sample project for algob). I think it could be integrated easily with `yarn unlink`

### CODE REVIEW

File: algobsetup [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    for (( i=0; i<${#LINKED_PACKAGES[@]}; i++ ));
+    do
+        link ${LINKED_PACKAGES[$i]}
+    done
+}
+
+
+check_dir_and_delete $ALGO_BUILDER_DIR
+git clone https://github.com/scale-it/algo-builder.git
```
[the remark this answers is not in the document]
Comment (Lumene98) [reply algobsetup:48]:
```suggestion
cd $ALGO_BUILDER_DIR
git pull
```
Yeah maybe it will be better to do something like this

File: algobsetup [hunk 2]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+BASEDIR=$(pwd)
```
Comment (robert-zaremba) [inline algobsetup:2]:
```suggestion
#!/usr/bin/env bash

# The scripts automates setup from source 
# It clones the algo-builder repository and uses `yarn link` to link
# all workspace packages (algob, runtime ...)

BASEDIR=$(pwd)
```

File: algobsetup [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+ALGO_BUILDER_DIR="algo-builder"
+declare -a PACKAGES=("algob" "runtime" "types-algosdk")
+declare -a LINKED_PACKAGES=("@algo-builder/algob" "@algo-builder/runtime" "@algo-builder/types-algosdk")
```
Comment (robert-zaremba) [inline algobsetup:5]:
```suggestion
declare -a LINKED_PACKAGES=("@algo-builder/web" "@algo-builder/algob" "@algo-builder/runtime" "@algo-builder/types-algosdk")
```

File: algobsetup (hunk 1 above)
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply algobsetup:53]: best will be: check if directory exists, then:
1. pull if yes
2. clone otherwise

**Comment by robert-zaremba** [comment]
@Lumene98  would you like to finish this PR?

**Comment by Lumene98** [comment]
> 
> 
> @Lumene98 would you like to finish this PR?

Sorry I completely forgot, this should do it!

File: algobsetup [hunk 4]
```diff
@@ -14,11 +14,18 @@ function err () {
   exit 1
 }
 
-function check_dir_and_delete(){
+function check_algob_dir(){
     if [[ -d $1 ]]
     then
-        echo "$1 already exists in this project, deleting it now..."
-        rm -rf $1    
+        echo "$1 already exists in this project..."
```
Comment (robert-zaremba) [inline algobsetup:20]:
```suggestion
        echo "$1 already exists in this project, puling the latest version"
```

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**